### ISSUE TITLE
Spring Web app fails when using `@RequestMapping.path` vs `value`

### ISSUE THREAD

**Issue by emmanuelbernard** [quarkusio/quarkus#13323]
```
@RestController
@RequestMapping(path ="/api" )
public class TodoResource {
    @GetMapping
    public String hello() {
        return "hello";
    }
}
```

Stacktrace:

```
java.lang.NullPointerException
    at io.smallrye.openapi.spring.ParameterProcessor.requestMappingValuesToPath(ParameterProcessor.java:1071)
    at io.smallrye.openapi.spring.SpringAnnotationScanner.processControllerClass(SpringAnnotationScanner.java:194)
    at io.smallrye.openapi.spring.SpringAnnotationScanner.processControllerClasses(SpringAnnotationScanner.java:169)
    at io.smallrye.openapi.spring.SpringAnnotationScanner.scan(SpringAnnotationScanner.java:121)
    at io.smallrye.openapi.runtime.scanner.OpenApiAnnotationScanner.scan(OpenApiAnnotationScanner.java:114)
    at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.generateAnnotationModel(SmallRyeOpenApiProcessor.java:409)
    at io.quarkus.smallrye.openapi.deployment.SmallRyeOpenApiProcessor.build(SmallRyeOpenApiProcessor.java:286)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
    at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:936)
    at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
    at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
    at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2046)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1578)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1452)
    at java.base/java.lang.Thread.run(Thread.java:834)
    at org.jboss.threads.JBossThread.run(JBossThread.java:479)
Resulted in: io.quarkus.builder.BuildException
```

**Comment by emmanuelbernard** [comment]
I am having openapi as well as a dependency

**Comment by geoand** [comment]
@emmanuelbernard can you please add the stacktrace?

**Comment by geoand** [comment]
@phillip-kruger I added the stacktrace and it looks like OpenAPI is causing the issue.

Mind taking a look?

**Comment by phillip-kruger** [comment]
I am on it !

**Comment by geoand** [comment]
Thanks 👍

**Comment by phillip-kruger** [comment]
This is fixed (https://github.com/quarkusio/quarkus/pull/13327) - maybe we can backport to still get the fix into 1.10.0 ? @gsmet ?
